doc/src/sgml/release-12.sgml

```diff
@@ -1174,16 +1553,23 @@ Branch: REL_11_STABLE [1785ac8ad] 2020-08-02 11:00:12 -0400
 Branch: REL_10_STABLE [e1b4135cf] 2020-08-02 11:00:12 -0400
 -->
      <para>
+<!--
       Fix corner case in decompression logic
       in <filename>contrib/pgcrypto</filename>'s
       <function>pgp_sym_decrypt</function> functions (Kyotaro Horiguchi,
       Michael Paquier)
+-->
+<filename>contrib/pgcrypto</filename>の<function>pgp_sym_decrypt</function>関数における圧縮の展開ロジックで稀な場合を修正しました。
```

Comment:
原文に記載されていないのが問題なのですが、稀にどうなるのかが分からないため補足が必要かと思います。

Author: Michael Paquier <michael@paquier.xyz>
Date:   Mon Jul 27 15:58:59 2020 +0900

    Fix corner case with 16kB-long decompression in pgcrypto, take 2
    
    A compressed stream may end with an empty packet.  In this case
    decompression finishes before reading the empty packet and the
    remaining stream packet causes a failure in reading the following
    data.  This commit makes sure to consume such extra data, avoiding a
    failure when decompression the data.  This corner case was reproducible
    easily with a data length of 16kB, and existed since e94dd6a.  A cheap
    regression test is added to cover this case based on a random,
    incompressible string.

上記がcommitログなので、

「<filename>contrib/pgcrypto</filename>の<function>pgp_sym_decrypt</function>関数における圧縮の展開ロジックで稀に不正なパケットを含めてしまう不具合を修正しました。」

とするのはどうでしょうか。

Reply:
「場合を修正」は少し変なので「稀な場合の**不具合を**修正しました。」とします。
次パラグラフに説明がありますし、翻訳を逸脱するので、「不正なパケット・・・」を加えるのは賛同できません。